doc/src/quickstart.md

```diff
@@ -45,6 +55,55 @@ fn main() -> Result<(), Box<std::error::Error>> {
 }
 ```
 
+In this example we are using `CustomAction` to write a simple calculator
+grammar that calculates user input as it is being parsed. We define lexing and
+grammar rules as follows, and store them in `calc.l` and `calc.y` respectively:
+
+```
+%%
+[0-9]+ "INT"
+\+ "PLUS"
+\* "MUL"
+\( "LBRACK"
+\) "RBRACK"
+[\t ]+ ;
+```
+
```

Comment:
Need something here otherwise people won't realise when the lex file stops and the grammar starts.